rune/libenclave/runelet.go

```diff
@@ -68,8 +68,16 @@ func StartInitialization(cmd []string, cfg *RuneletConfig) (exitCode int32, err
 		}
 
 		// Launch a remote attestation to the enclave runtime.
-		if err = rt.LaunchAttestation(); err != nil {
-			return 1, err
+		product := "development"
+		quote_type := "SGX_UNLINKABLE_SIGNATURE"
+		if config.RaEpidQuoteType == "linkable" {
+			quote_type = "SGX_LINKABLE_SIGNATURE"
+		}
+
+		if config.RaType == "EPID" && config.RaEpidSpid != "" && config.RaEpidSubscriptionKey != "" {
+			if err = rt.LaunchAttestation(config.RaEpidSpid, quote_type, product, config.RaEpidSubscriptionKey); err != nil {
```

Comment:
The type of quote_type would be integer not string. The type of variable product should be bool.

Reply:
> The type of quote_type would be integer not string. The type of variable product should be bool.

Add `product` to an optional field `Spec.annotations` of config.json is OK, but it must be `string` type.